replica/tablets.hh

```diff
@@ -28,8 +31,13 @@ class query_processor;
 
 namespace replica {
 
+static thread_local auto replica_type = tuple_type_impl::get_instance({uuid_type, int32_type});
+static thread_local auto replica_set_type = list_type_impl::get_instance(replica_type, false);
```

Comment:
If you define these in a header then each compilation unit will get its own instance, which is wasteful. Better keep them defined in tablets.cc and make them extern here, or add functions to get them.